epsagon/trace.py

```diff
@@ -483,6 +492,7 @@ def __init__(
         propagate_lambda_id=False,
         logging_tracing_enabled=False,
         step_dict_output_path=None,
+        sample_rate=1,
```

Comment:
why the default both here on Trace creation and on TraceFactory?

Reply:
1. as you can see - all parameters are being set to defaults both in Trace and TraceFactory.
It's probably not ideal, as well as it's not ideal to hand over more than 20 parameters to a constructor, but it is how it is for now.
2. There is at least one more place that initialises Trace, besides TraceFactory. Won't the the chance to break anything.
It's just a default value.

Reply:
Opened a ticket to look into refactoring that